README.md

```diff
@@ -2,22 +2,18 @@
 
 <img src="stretchly_128x128.png" align="right" alt="Stretchly logo">
 
-> **The break time reminder app**
-
-*Stretchly* is a cross-platform [Electron](https://www.electronjs.org/) app that reminds you to take breaks when working on your computer.
+*Stretchly* is a cross-platform [Electron](https://www.electronjs.org/) app that reminds you to take breaks while working at the computer.
 
 [![Become a Sponsor!](https://img.shields.io/static/v1?label=Sponsor&message=%E2%9D%A4&logo=GitHub&color=success)](https://github.com/sponsors/hovancik) [![Become a Patron!](https://img.shields.io/static/v1?label=Sponsor&message=%E2%9D%A4&logo=Patreon&color=success)](https://www.patreon.com/hovancik)
 
 ## Table of contents
 - [Install](#install--)
 - [Default behavior](#default-behavior)
 - [Preferences](#preferences)
-- [Advanced Preferences](#advanced-preferences)
```

Comment:
I would like to keep this.

Reply:
It stays, I just made them subsections so it does not make much sense for it to show up in the ToC, no?
I feel like the ToC was a little overwhelming before.

Reply:
You can still link to them, even if they don't appear in the ToC ;)